core/transaction.go

```diff
@@ -205,6 +268,18 @@ type InvokeTransaction struct {
 	Nonce *felt.Felt
 	// The address of the sender of this transaction
 	SenderAddress *felt.Felt
+
+	// Version 3 fields (there was no version 2)
+	ResourceBounds map[Resource]*ResourceBounds
```

Comment:
any particular reason these are pointers?

```suggestion
	ResourceBounds map[Resource]ResourceBounds
```

Reply:
No.